### DISCUSSION TITLE
Media Content Getting Lost on Subsequent Deployment

### DISCUSSION THREAD

**Discussion by aadharjain1** [strapi/strapi#8306] [Legacy Topics]
I am deploying Strapi using Docker on an Azure instance, but on subsequent deployment all my previous media content is getting lost. Is there a way to persist data on the server?

**Comment by MattieBelt** [comment]
Hey @aadharjain1, 

I see you are wanting to ask a question, Strapi is currently planning to decomission the GitHub Discussions in early 2021 and they don't want to lose the great questions asked by our community.

Questions should be directed to the [forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

Your question is simple and I want to redirect you to this related [FAQ](https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#why-is-my-app-s-database-and-uploads-resetting-on-heroku), so you should choose an [upload provider](https://strapi.io/documentation/v3.x/plugins/upload.html#using-a-provider) for it.

**Reply by aadharjain1** [reply]
Thanks for the help @MattieBelt.
